src/Controllers/EntityBrowserFormController.php

```diff
@@ -68,17 +83,10 @@ public static function create(ContainerInterface $container) {
   }
 
   /**
-   * Test implementation of standalone entity browser page.
+   * {@inheritdoc}
    */
-  public function page() {
-    $browser = $this->loadBrowser();
-
-    // The original path is sometimes needed: ie for views arguments.
-    if ($original_path = $this->request->get('original_path')) {
-      $browser->addAdditionalWidgetParameters(['path_parts' => explode('/', $original_path)]);
-    }
-
-    return $this->entityFormBuilder()->getForm($browser, 'entity_browser');
+  protected function getFormObject(RouteMatchInterface $route_match, $form_arg) {
```

Comment:
We don't, but we need to keep function's signature since we're overriding parent function.